src/inspect_ai/_view/www/README.md

```diff
@@ -24,5 +24,7 @@ Use the following commands (run in the `src/inspect_ai/_view/www` dir) to ensure
    ```bash
    yarn build
    ```
-   
-   Don't forget to stage newly updated changes in the `dist` folder.
\ No newline at end of file
+
+   Don't forget to stage newly updated changes in the `dist` folder.
+
+   You may optionally set the `API_URL` environment variable at build time to use an API server running on a different host.
```

Comment:
One thing I'm thinking about it is that we may want to help clean up the current naming by doing something like `api-browser` -> `api-view-server` (current the `http` vs `browser` distinction isn't really clear). Given this + the fact that this parameter doesn't really affect the other api clients, thinking we should scope the name like `VIEW_SERVER_API_URL` or `SERVER_API_URL`.

Reply:
Sure, done!